### DISCUSSION TITLE
Highly available MinIO setup with common centralized data persistence.

### DISCUSSION THREAD

**Discussion by kushwaharishi** [minio/minio#13640] [Q&A] [answered]
Hello,
My name is Rishi Kushwaha, I am a developer working for government department in Sydney.
We are looking for a highly available MinIO setup for unstructured data storage, but with a common centralized persistence.
Could you please suggest the mode and configuration we should be using for this requirement.

**Comment by klauspost** [comment, marked as answer]
For availability you should definitely go for a [distributed setup](https://docs.min.io/minio/baremetal/installation/deployment-and-management.html).

The actual sizes and similar depends too much on your requirements to make recommendations. You can write `hello@minio.io` and we can assist you directly in designing and setting up your system.
